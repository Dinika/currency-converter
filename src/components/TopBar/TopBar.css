.ui.menu {
  box-sizing: border-box;
  width: 100vw;
  background: #2e303d;
  position: fixed;
  top: 0px;
  height: 100px;
  border-radius: 0;
  display: flex;
  justify-content: center;
}

.ui.menu > .item {
  font-family: Poppins, Lato, Arial, sans-serif;
  font-size: 2em;
  font-weight: 800;
  color: #f8fbfd;
}

.ui.header {
  color: #8c888e;
  font-family: Lato;
}
