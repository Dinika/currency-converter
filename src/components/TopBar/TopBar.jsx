import React from 'react';
import { Menu } from 'semantic-ui-react';
import classes from './TopBar.module.css';

const TopBar = props => (
  <Menu className={classes.TopBar}>
    <Menu.Item name="appTitle">{props.title}</Menu.Item>
  </Menu>
);

export default TopBar;
