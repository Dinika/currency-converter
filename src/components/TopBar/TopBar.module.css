.TopBar {
  background-color: #2e303d;
}
