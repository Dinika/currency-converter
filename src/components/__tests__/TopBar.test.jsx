import React from 'react';
import { shallow } from 'enzyme';
import TopBar from '../TopBar/TopBar';

describe('TopBar', () => {
  it('renders with all props set', () => {        
    const wrapper = shallow(
        <TopBar title="Some Title" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});