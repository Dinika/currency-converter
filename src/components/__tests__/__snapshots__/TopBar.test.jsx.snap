// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar renders with all props set 1`] = `
<Menu
  className="TopBar"
>
  <MenuItem
    name="appTitle"
  >
    Some Title
  </MenuItem>
</Menu>
`;
